### PR TITLE
Azure DNS new zone example template.

### PR DESCRIPTION
**Pull request by GarethBradshawMSFT**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@GarethBradshawMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (garbrad). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: azure-dns-new-zone/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newZoneName": {
+      "type": "String",
+	  "defaultValue": "example.com",
```
Comment (singhkay) [inline azure-dns-new-zone/azuredeploy.json]: Does this parameter need to be unique?

File: azure-dns-new-zone/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+  },
+  "resources": 
+  [
+    {
+      "type": "microsoft.network/dnszones",
+      "name": "[parameters('newZoneName')]",
+      "apiVersion": "2015-05-04-preview",
+      "location": "global",
```
Comment (singhkay) [inline azure-dns-new-zone/azuredeploy.json]: Is global the only acceptable value for location?

**Comment by singhkay** [comment]
@GarethBradshawMSFT Left some comments

File: azure-dns-new-zone/azuredeploy.json (hunk 1 above)
Comment (GarethBradshawMSFT) [inline azure-dns-new-zone/azuredeploy.json]: We can support multiple copies of the same DNS zone but it's limited, so I can remove the default if desired.

File: azure-dns-new-zone/azuredeploy.json (hunk 2 above)
Comment (GarethBradshawMSFT) [inline azure-dns-new-zone/azuredeploy.json]: Yes, all DNS resources are "global".

**Pull request merged**